browse/factory.py

```diff
@@ -13,10 +14,12 @@
 from arxiv.base import Base
 from arxiv.users.auth import Auth
 
+# s3 = FlaskS3()
+
 
 def create_web_app() -> Flask:
     """Initialize an instance of the browse web application."""
-    app = Flask('browse', static_folder='static', template_folder='templates')
+    app = Flask('browse', static_url_path='/static')
```

Comment:
What happened to the template parameter?

Reply:
`templates` is the default value, so that change shouldn't affect anything. I'm still messing a bit with flask-s3, so am not quite ready yet.

Reply:
The legacy system has a problem where the static assets are difficult to develop with. 
It would be great only the production system served assets from S3.